main.nf

```diff
@@ -80,6 +80,36 @@ process picard_collectrnaseqmetrics{
 
 }
 
+process stringtie{
+
+    input:
+        tuple val(sample) , file(bam),file(bai)  from stringtie_input
+
+    output:
+        tuple val(sample), file("${sample}.stringtie.gtf") into stringtie_output
+
+    script:
+
+    """
+    stringtie ${bam} -G ${params.gtf} > ${sample}.stringtie.gtf
```

Comment:
Should we add the strandedness of the library? We could also add option `-p` to allocate more cpus for the assembly

Reply:
the deed is done.